crates/matrix-sdk-crypto/src/store/mod.rs

```diff
@@ -1490,13 +1497,14 @@ impl Store {
         exported_keys: Vec<ExportedRoomKey>,
         progress_listener: impl Fn(usize, usize),
     ) -> Result<RoomKeyImportResult> {
-        self.import_room_keys(exported_keys, None, progress_listener).await
+        self.import_room_keys(exported_keys, None, None, progress_listener).await
     }
 
     async fn import_sessions_impl<T>(
         &self,
         room_keys: Vec<T>,
         from_backup_version: Option<&str>,
+        sender_data: Option<&SenderData>,
```

Comment:
given that this ends up as the `forwarder_data` rather than the `sender_data`, it should probably be named accordingly.